doc/src/manual/code-loading.md

```diff
@@ -190,26 +206,32 @@ paths = Dict{Tuple{UUID,Symbol},String}(
 )
 ```
 
-This example map includes three different kinds of package locations:
+This example map includes three different kinds of package locations (the first and third are part of the default load path):
 
 1. The private `Priv` package is "[vendored](https://stackoverflow.com/a/35109534/659248)" inside the `App` repository.
 2. The public `Priv` and `Zebra` packages are in the system depot, where packages installed and managed by the system administrator live. These are available to all users on the system.
 3. The `Pub` package is in the user depot, where packages installed by the user live. These are only available to the user who installed them.
 
+
 ### Package directories
 
-Package directories provide a kind of environment that approximates package loading in Julia 0.6 and earlier, and which resembles package loading in many other dynamic languages. The set of packages available in a package directory corresponds to the set of subdirectories it contains that look like packages: if `X/src/X.jl` is a file in a package directory, then `X` is considered to be a package and `X/src/X.jl` is the file Julia loads to get `X`. Which packages can "see" each other as dependencies depends on whether they contain project files, and if they do, on what appears in those project files' `[deps]` sections.
+Package directories provide a simpler environment, but less powerful or controlled. The set of root packages available is defined simply as the set of files in the directory that "look like" packages. A package `X` is determined to exist if it finds one of the following files:
+  - A file `X.jl`
+  - A file `X/src/X.jl`
+  - A file `X.jl/src/X.jl`
+Additionally, which packages it is able to import as dependencies depends on whether that package contains a project file. If it does not, it can import any root package. Otherwise (if it does contain a `Package.toml` file), it can only import those packages which are identified in its `[deps]` sections (given by uuid and name).
```

Comment:
```suggestion
Additionally, which dependencies a package in a package directory is able to import depends on whether the package contains a project file:
* If it does not have a project file, it can import any root package—the same packages that can be loaded at the top level from `Main` and in the REPL.
* If it does have a project file, it can only import those packages which are identified in the `[deps]` section of the project file.

This dependency loading is part of the `graph` map for package directories, explained elsewhere.
```